src/main/java/org/eclipse/xpanse/terraform/boot/terraform/TerraformExecutor.java

```diff
@@ -72,7 +72,8 @@ public TerraformExecutor(SystemCmd systemCmd,
             this.moduleParentDirectoryPath =
                     System.getProperty("java.io.tmpdir");
         } else {
-            this.moduleParentDirectoryPath = moduleParentDirectoryPath;
```

Comment:
we do not need this change. This must be controlled fully by the var. 
If the path is /tmp/xpanse_ws_dir then we must pass this property directly in the server startup.